### ISSUE TITLE
Citation

### ISSUE THREAD

**Issue by miyamot0** [nrobb/posm#1]
Hey Nigel,

You ever publish anything with this code? I have a simple demo alongside a submission and I was gonna derive a basic online learning example using this algorithm. 

You have a DOI?

**Comment by nrobb** [comment]
Hi Shawn ,
I didn't publish anything yet. I have a paper in progress on some research using this in a simple game, but haven't got it accepted anywhere so far.

I haven't looked at this code in a long time, it's probably very buggy lol

**Comment by miyamot0** [comment]
Lol, I did some preliminary simulations in several hypothetical use cases for an instructional task. POSM is pretty wide out of the gate, which is a bit out of the ordinary in an instructional context, but it settled pretty well against a 'true' hidden ability in simulated work. My work is still pretty early, but I think we're both probably looking for the type of method.

**Comment by nrobb** [comment]
Yeah, I think so. I didn't get that far in this area tbh, but I'm still very interested in it. Keep me updated if you make any progress!

**Comment by miyamot0** [comment]
Also, do you have an email for followup (or ideally, a slack account)? 

I have a few game-based studies planned in the future and would appreciate your thoughts.
